elements/xml_form_elements.module

```diff
@@ -303,11 +303,35 @@ function xml_form_elements_form_element_tabs_ajax_add(FormElement $element, arra
     if ($element->controls['#type'] == 'tags') {
       foreach ($element->children as $child) {
         $child->default_value = $child->getOriginalDefaultValue();
-        if (empty($child->default_value)) {
+
+        // We should orphan object only if they have no default AND it is not
+        // the last child.  We must retain one, otherwise we'll get AJAX an
+        // AJAX error when adding an element after adding another tab.
+        $no_default = empty($child->default_value);
+        $is_not_last = count($element->children) > 1;
+
+        if ($no_default && $is_not_last) {
           $child->orphan();
         }
       }
+
+      // In order to provide functionality identical to when the tab is first
+      // created, we need to dedeupe based on the first element's default
+      // value.  Otherwise we can wind up with multiples of the same default
+      // value.
```

Comment:
Actually... yeah... all of this function seems to be dealing with symptoms, instead of addressing the issue. It seems that the problem is that we're dealing with [a clone from the _current_ form state](https://github.com/Islandora/islandora_xml_forms/pull/214/files#diff-9fd1f16ddbdc498ecc545353130d9253R300), instead of pulling in a fresj copy from the original form definition, which would(/should) have the original defaults intact.